### DISCUSSION TITLE
Installing Waripinator in (X)ubuntu 18.04

### DISCUSSION THREAD

**Discussion by jarnos** [linuxmint/discussions#591] [Questions]
`README.md` tells that once I add GRPC PPA I could install using instructions for Ubuntu 20.04, but I can not install `python3-grpcio` for Bionic. So can you install Warpinator in 18.04?

**Comment by anaximeno** [comment]
You can try to install it through flatpak https://sites.google.com/site/installationubuntu/home/ubuntu-17-10/flatpak (flatpak installation instruction for ubuntu 18.04)
